docker-compose.yml

```diff
@@ -27,6 +34,8 @@ services:
     airflow-scheduler:
         build:
             context: ./airflow
+            args:
+                IMAGE_TAG: ${IMAGE_TAG}
```

Comment:
to avoid the duplication of all `airflow-*` images, maybe we should have a `airflow` one with `build:` and `image:` that all the others depend upon. The other can then use just `image:` and the `build:` configuration is in a single place